dev/docker/build.sh

```diff
@@ -1,4 +1,21 @@
 #!/bin/bash
+
+while getopts ":rpi:" opt; do
```

Comment:
I was thinking more along the lines of using `"$@"` in the `docker build` line than full on argument parsing, but I guess this could work.